javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,215 @@
+
+       /************************************************** 
+      *       1  :  Flight booking fullname function      *
+       ***************************************************/  
+
+           console.log('(1a) : Printing out full name by using returne \n\n ');
+
+     /*************************************************** 
+    * (1a) : Printing out full name by using return      *
+     ****************************************************/  
+
+
+
+
+var firstName='';
+let lastName='';
+function getFullName(firstName,lastName){
+  return firstName+' '+lastName;
+}
+let fullName=getFullName("Benjamin", "Hughes");
+console.log(fullName);
+
+
+console.log('\n\n (1b) :       Formal fullname  \n\n ');   
+
+
+    /********************************************* 
+    *  (1b) :       Formal fullname               *
+     *********************************************/   
+           
+
+let useFormalName = true;
+function getFullName(firstName,lastName,useFormalName){
+  if(useFormalName)
+  return 'Lord'+' '+ firstName+' '+lastName;
+  else 
+  return firstName+' '+lastName;
+}
+fullName=getFullName('Benjamin', 'Hughes',false);
+console.log(fullName);
+let formalName=getFullName('Benjamin', 'Hughes',true);
```

Comment:
always use const until it doesn't let you

Reply:
yea Noted again :)